tests/integration/AbstractHttpApiServerTestCase.php

```diff
@@ -72,14 +72,13 @@ protected function setUp(): void
         $this->client = new Client();
 
         // wait for server to start
-        usleep(200000);
+        usleep(100000);
```

Comment:
What do yo think about actually testing, obviously if on an overloaded server 100ms might be a little touchy.

https://gist.github.com/bweston92/e31d35229499b3fa5b0f9cea95a93fd7#file-test_socket_to_stop_risky_test-php-L44-L58